docs/usage.rst

```diff
@@ -224,6 +224,29 @@ The one exception to this rule is for trigonometric functions applied to data wi
   >>> np.sin(45.*degree)
   array(0.70710678)
 
+Logarithmic Quantities and Units
+********************************
+
+The logarithmic quantities level-of-power and level-of-field and the units neper and
+bel are supported. In the next example, we represent the power measurements, p, as
+a logarithmic quantity at reference level, p_ref, in the units decibel.
+
+  >>> import numpy as np
+  >>> from unyt import dB, mW
+  >>> dB.dimensions
+  (logarithmic)
+  >>> p = [1, 100]*mW
+  >>> p_ref = 1*mW
+  >>> level_of_power = 10*np.log10(p/p_ref)*dB
+  >>> level_of_power
+  unyt_array([ 0., 20.], 'dB')
+
+You can convert the logarithmic quantity back to physical units through exponentiation,
+just remember to remove the units using the unyt_array.v property.
```

Comment:
`unyt_array.v` needs either double backticks or, even better, to be a cross-reference to the API docs.

Reply:
done.